app/products/calls/components/emoji_list.tsx

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {StyleSheet, View} from 'react-native';
+import LinearGradient from 'react-native-linear-gradient';
+
+import EmojiPill from '@calls/components/emoji_pill';
+import {ReactionStreamEmoji} from '@calls/types/calls';
+
+const styles = StyleSheet.create({
+    container: {
+        position: 'relative',
+        width: '100%',
+        height: 48,
+    },
+    emojiList: {
+        display: 'flex',
+        flexDirection: 'row',
+        alignItems: 'center',
+        justifyContent: 'flex-start',
+        paddingLeft: 10,
+        paddingRight: 10,
+        height: '100%',
+    },
+    gradient: {
+        position: 'absolute',
+        top: 0,
+        bottom: 0,
+        width: '100%',
+    },
+});
+
+interface Props {
+    reactionStream: ReactionStreamEmoji[];
+}
+
+const EmojiList = ({reactionStream}: Props) => {
+    return (
+        <View style={styles.container}>
+            <View style={styles.emojiList}>
+                {reactionStream.map((e) => (
```

Comment:
I didn't see a limit set to the reactionStream in terms of different emojis, should we set a limit to avoid too many of them being rendered off screen?

Reply:
Good idea -- I set it to max I can see on an iPad, plus two extra just in case.